clients/src/main/java/org/apache/kafka/clients/consumer/WakeupException.java

```diff
@@ -14,7 +14,7 @@
 
 import org.apache.kafka.common.KafkaException;
 
-public class ConsumerWakeupException extends KafkaException {
+public class WakeupException extends KafkaException {
```

Comment:
Can we move this out of the `consumer` package? This is used in Copycat as well.